### ISSUE TITLE
Cannot run due to some LANfilters?

### ISSUE THREAD

**Issue by LiamDawe** [opennox/opennox#498]
Hi, trying to run it for the first time today on Fedora Linux using the Snap, eventually figured out the file needed but I'm coming up against this issue when trying to run it:

```
[liam@desktop-f7tu9t7 ~]$ snap run opennox
2022/06/20 12:28:05 [version]: version: v1.8.10 ()
2022/06/20 12:28:05 [version]: GET https://ghcr.io/token?scope=repository%3Anoxworld-dev%2Fopennox%3Apull
2022/06/20 12:28:05 [config]: using file: "/home/liam/snap/opennox/common/opennox.yml"
2022/06/20 12:28:05 [path]: setting data dir to: "/home/liam/Games/gog/nox/drive_c/GOG Games/Nox"
2022/06/20 12:28:05 [sdl]: window size: 640x480
2022/06/20 12:28:05 [sdl]: OpenGL version: 3.3.0 NVIDIA 510.68.02
2022/06/20 12:28:05 [render]: creating surface: 640x480
2022/06/20 12:28:05 [game]: game state code: Movies
2022/06/20 12:28:05 game flag set: Host | Client
2022/06/20 12:28:05 game flag set: Arena
2022/06/20 12:28:06 [config]: reading legacy: "nox.cfg"
2022/06/20 12:28:06 [config]: reading legacy: "default.cfg"
2022/06/20 12:28:06 [version]: GET https://ghcr.io/v2/noxworld-dev/opennox/tags/list
failed to load config file: cannot parse LANFilters: invalid filter values: "0,0,0,0,0,250,0,0,0,0,0"
```

**Comment by lucifertdark** [comment]
I had the same thing just now, I solved it by opening the default.cfg file & changing the line "LANFilters = 0,0,0,0,0,250,0,0,0,0,0,0" to read "LANFilters = 0,0,0,0,0,0,0,0,0,0,0,0". the game runs perfectly after that.

**Comment by dennwc** [comment]
Sorry, it was a regression in v1.8.10, which was fixed in ebcda928. Will be included in the next release. For now either changing config or rebuilding from source will work.
